cloudflare/provider.go

```diff
@@ -100,6 +101,7 @@ func Provider() terraform.ResourceProvider {
 			"cloudflare_page_rule":              resourceCloudflarePageRule(),
 			"cloudflare_rate_limit":             resourceCloudflareRateLimit(),
 			"cloudflare_record":                 resourceCloudflareRecord(),
+			"cloudflare_spectrum_app":           resourceCloudflareSpectrumApp(),
```

Comment:
I think we should use the full word "application" here and for all other references too.

```suggestion
			"cloudflare_spectrum_application":           resourceCloudflareSpectrumApplication(),
```